txclib/exceptions.py

```diff
@@ -37,3 +39,11 @@ class HttpNotAuthorized(Exception):
 
 class AuthenticationError(Exception):
     pass
+
+
+class ConnectionError(ConnectionError):
```

Comment:
I would suggest to rename the custom class to something more specific e.g. `TXConnectionError` so that we avoid confusion.